src/xenia/base/exception_handler_win.cc

```diff
@@ -40,9 +40,9 @@ LONG CALLBACK ExceptionHandlerCallback(PEXCEPTION_POINTERS ex_info) {
   thread_context.rip = ex_info->ContextRecord->Rip;
   thread_context.eflags = ex_info->ContextRecord->EFlags;
   std::memcpy(thread_context.int_registers, &ex_info->ContextRecord->Rax,
-              sizeof(thread_context.int_registers));
+              sizeof(*thread_context.int_registers));
```

Comment:
@drillsar Information output from tools such as PVS Studio are meant to be guidance with regards to warnings/errors about code. It cannot perfectly understand every scenario especially in a project such as Xenia.